medusa/clients/torrent/utorrent_client.py

```diff
@@ -135,7 +201,8 @@ def _set_torrent_priority(self, result):
 
     def _set_torrent_pause(self, result):
         return self._request(params={
-            'action': 'pause' if app.TORRENT_PAUSED else 'start',
+		# Rafi: "pause" changed to "stop" since pause is unhealthy to the swarm, and now removed from uTorrent toolbar...
```

Comment:
This should still pause since that's what the function is for.

Reply:
Sure, I'll  remove the comments, these were just for helping to clear the changes for your review ... Do you want me to remove all of them or just the "Rafi:" text in them?

The replacement of "pause" to "stop" is intentional! I am a beta tester for uTorrent for a long time, and as I tried to explain, pause is now not recommended and removed from the uTorrent toolbar (I'll not go into details). "Stop" sill has  the exact same effect as "pause", so no change in functionality here. The user can resume/re-start the download job at any time. I am aware that the text on the GUI can be changed as well, but I assumed it does not matter much. Let me know if you want me to change the text on the GUI checkbox as well.

Regarding re-basing, sorry, but I really don't know what/how to do this, so if you can help? This is what I see on my side, as if all commits are already on develop branch :( 
http://www.imagebam.com/image/ee5ec5823517443

Thanks!

PS: will you be interested in some ideas on how to add a simple "smart-file-size" filter, so users will be able to specify minimum file size for their downloads?

Reply:
We tend to use Git commit messages and comments on here to keep notice of what's happening.

Reply:
HI,
Let me know if you can you handle this one PR as  is (on develop while coming form master on my end) or really need me to copy everything to another branch on my side, which is a bit of a hassle... 

>For example a HDTV rip will likely be about 1GB per episode in H264 where as it'll be around 250MB in HEVC.
^ this is also a good example  for my idea/use-case: specifying minimum file size in your example will be a good tool for the user to specify that he  does not like, for example, compressed x265 files (for less cpu % and better quality) .

See below - another example, where specifying 720p can snatch smaller files, when the user wants the best 720p quality, like with good 5.1 audio, when the smaller files can be 2.0 channels.

![1](https://user-images.githubusercontent.com/2458280/39031445-d791192a-4470-11e8-8a18-7c706fd9050c.png)